opencti-platform/opencti-front/src/private/components/analysis/Reports.tsx

```diff
@@ -63,45 +80,6 @@ const Reports: FunctionComponent<ReportsProps> = ({
     'view-reports',
   );
 
-  const { reportType } = match.params;
-
-  const reportFilterClass = reportType !== 'all' && reportType !== undefined
-    ? reportType.replace(/_/g, ' ')
-    : '';
-
-  const additionnalFilters = [];
-  if (reportFilterClass) {
```

Comment:
Why are you removing that ?

Reply:
I only removed the reportType part, the additionalFilters are above now.
The reportType is useless as it is part of the URL and filters and that's exactly what useLocalStorage is designed for, so it's redundant.